src/public.ts

```diff
@@ -59,7 +59,7 @@ window.addEventListener('DOMContentLoaded', () => {
 	}
 
 	// Adding double-download warning
-	const downloadButtons = document.querySelectorAll<HTMLAnchorElement>('a[href*="/download/"]') || []
+	const downloadButtons = Array.from(document.querySelectorAll<HTMLAnchorElement>('a[href*="/download/"]'))
```

Comment:
`querySelectorAll` does not return an array but a `NodeList`, so passing it to a `Set` works only if the browser likes you :sweat_smile: